packages/design/src/tokens/dark.tokens.json

```diff
@@ -51,7 +51,7 @@
             }
         },
         "critical": {
-            "$value": "{color.base.red-.600}"
+            "$value": "{color.base.red-.500}"
```

Comment:
This is the key change here - destructive and critical now match

![image](https://github.com/user-attachments/assets/cda457a8-dca5-4503-a934-b7ceb26830ae)